### PR TITLE
Comparing Intrusion Detection Systems (IDS) and Intrusion Prevention Systems (IPS)

### LINKED ISSUE #841: Comparing Intrusion Detection Systems (IDS) and Intrusion Prevention Systems (IPS)
### Brief Summary: 
Describe the what, why, and how of your content idea in 2-5 sentences.

A network intrusion is any unauthorized activity on a computer network. Intrusion Detection Systems (IDS) and Intrusion Prevention Systems (IPS) can help block intruders who aim to steal sensitive data, cause data breaches, and install malware.
IIDS analyze network traffic for signatures that match known cyber-attacks. IPS also analyze packets and stops the packet from being delivered based on the attacks it detects, helping stop the attack.

### Key Takeaways: 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?

1. An overview of IDS and IPS 
2. Problems IDS/IPS Address 
3. IDS vs. IPS
4. Why IDS and IPS are Critical for Cybersecurity
### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)
N/A

### PR DESCRIPTION
**Pull request by EKahuha**
this fixes #841

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by hectorkambow** [comment]
👋 Thank you for submitting this PR - we will review it ASAP - all PRs go into our queue will be looked over before being approved to be published.

File: articles/comparing-intrusion-detection-systems-ids-and-intrusion-prevention-systems-ips/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+| Not inline, thus traffic does not have to flow through IDS | Traffic must flow through IPS |
+| False positives for IDS cause alerts | False positives for IPS may lead to loss of functions or data |
+| IDS act as monitoring systems | IPS act as control systems |
+| Do not manipulate network traffic | IPS restrains packets from delivering depending on the contents of the packet |
+
+### Why IDS and IPS are critical for cybersecurity
+
+Data breach threats are ever-growing, and security teams must continuously update their cybersecurity strategies. IDS and IPS are essential aspects of a cybersecurity strategy and can help to accomplish many security objectives, as explained below.
+
+**Compliance**: in many jurisdictions, part of the compliance requirement is to invest in systems and technologies that guarantee data protection. These technologies are not limited to IPS and IDS, which are significant in addressing security controls. Security controls act as countermeasures or safeguards to detect, avoid, minimize, and counteract threats to computer systems, data, computer networks, and more.
```
Comment (lalith1403) [inline articles/comparing-intrusion-detection-systems-ids-and-intrusion-prevention-systems-ips/index.md:59]: Capitalize the first word in the sentence.

Comment (lalith1403) [reply articles/comparing-intrusion-detection-systems-ids-and-intrusion-prevention-systems-ips/index.md:59]: in -> In

File: articles/comparing-intrusion-detection-systems-ids-and-intrusion-prevention-systems-ips/index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+**Policy enforcement**: security teams can configure IDS and IPS to implement internal security strategies at the network level. If an organization supports only one virtual private network (VPN), it can rely on IPS to block other VPN traffic.
```
Comment (lalith1403) [inline articles/comparing-intrusion-detection-systems-ids-and-intrusion-prevention-systems-ips/index.md:61]: security -> Security

File: articles/comparing-intrusion-detection-systems-ids-and-intrusion-prevention-systems-ips/index.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+**Automation**: since IPS and IDS are mostly hands-off, they are very applicable for the present security stack. IPS only requires limited resources to protect networks from known threats.
```
Comment (lalith1403) [inline articles/comparing-intrusion-detection-systems-ids-and-intrusion-prevention-systems-ips/index.md:63]: since -> Since

**Review by lalith1403 — APPROVED** [review]
The article is well written and informative. Very few minor changes were made. Please make a note of them.

**Comment by hectorkambow** [comment]
plagiarism check done

**Pull request merged**